src/Pages/Dashboard/Sensor.jsx

```diff
@@ -50,8 +122,15 @@ class Sensor extends PureComponent {
       <div className={classes.root}>
         <Dashboard
           data={this.state.data}
+          historyData={this.state.historyData}
           sensorName={this.props.sensorName}
           sensorId={this.props.sensorId}
+          changeRange={this.changeRange}
+          changeCalStart={this.handleStartDateTimeChange}
+          changeCalEnd={this.handleEndDateTimeChange}
+          selected={this.state.selected}
+          start={this.state.start}
```

Comment:
Is this necessary? I believe I did all these just to eliminate the errors, so I do not have a good reason to do it that way.